### ISSUE TITLE
QueueTrigger RTS improvements

### ISSUE THREAD

**Issue by mathewc** [Azure/azure-webjobs-sdk#2640]
We should consider adding the new QueueTrigger scale heuristics that were added to ScaleController to the RTS logic. These ScaleController changes were made after the initial version of RTS was released.

Those ScaleController changes also apply to ServiceBus, so if we do this we should also do it for ServiceBusTrigger.

**Comment by v-anvari** [comment]
Hi @mathewc , Can you confirm if this issue still requires a triage or can be closed

**Comment by mathewc** [comment]
Yes, this work item is still relevant. We'll need to decide at this point whether we only make the improvements to the Track 2 extensions (likely).

**Comment by alrod** [comment]
this needs to be done by functions team.
[3:44 PM] Vincent Chiang
So in scale controller, we have code that helps burst scale sb queues and storage queues (if the msg has been in queue > 1.5s) https://msazure.visualstudio.com/One/_git/AAPT-Antares-ScaleController?path=/src/Microsoft.Azure.AppService.ScaleController.Extension/ScalerBase.cs&version=GBv2.x&line=323&lineEnd=325&lineStartColumn=13&lineEndColumn=63&lineStyle=plain&_a=contents

[3:44 PM] Vincent Chiang
However, this code has not been ported over to the storage queue or sb queue implementations: 
https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/Queues/QueueListener.cs#:~:text=private%20ScaleStatus-,GetScaleStatusCore,-(int%20workerCount
azure-sdk-for-net/QueueListener.cs at main · Azure/azure-sdk-for-net
This repository is for active development of the Azure SDK for .NET. For consumers of the SDK we recommend visiting our public developer docs at https://docs.microsoft.com/dotnet/azure/ or our vers...

Moving to azure-sdk-for-net as tha changes will be implemented in track2
